src/Files.App/ViewModels/ShellViewModel.cs

```diff
@@ -2496,6 +2496,7 @@ private async Task AddFileOrFolderAsync(ListedItem? item)
 				else if (storageItem.IsOfType(StorageItemTypes.Folder))
 				{
 					var properties = await storageItem.AsBaseStorageFolder().GetBasicPropertiesAsync();
+					size = item.IsArchive ? (long)properties.Size : null;
```

Comment:
```suggestion
					size = (long)properties.Size;
```
This should do the trick

Reply:
GetBasicPropertiesAsync only returns the size for files, so it doesn't need to be conditional.

Reply:
On second thought, it seems to depend on the folder. I've added back the check just in case.

Reply:
That was the issue I had as well. In any case, I added back the check and it's working as expected.